scripts/readme/workflow_generator.py

```diff
@@ -98,7 +98,7 @@ def write_notebook_workflow(notebook, name, output_telemetry=Telemetry()):
     output_telemetry.gh_working_dir = gh_working_dir
 
 
-def write_workflows(notebooks, output_telemetry=[]):
+def write_workflows(notebooks, output_telemetries=[]):
```

Comment:
Nit: use None as default value since mutable default value will be changed during function execution.

Reply:
this is worked as an output value of write_workflows.

I insert this parameter everywhere to collect data for generate README.md

Reply:
Updated.